lhotse/dataset/speech_recognition.py

```diff
@@ -80,30 +77,20 @@ def __init__(
             Examples: normalization, SpecAugment, etc.
         :param input_strategy: Converts cuts into a collated batch of audio/features.
             By default, reads pre-computed features from disk.
-        :param check_inputs: Should we iterate over ``cuts`` to validate them.
-            You might want to disable it when using "lazy" CutSets to avoid a very long start up time.
         """
         super().__init__()
         # Initialize the fields
-        self.cuts = cuts
```

Comment:
I am trying to print the decoding progress using
`batch_idx / len(self.cuts) * 100`

But `self.cuts` is removed in this pull-request. Is there a way to get the total number of cuts in a `K2SpeechRecognitionDataset`?

Reply:
In specific cases, yes. You can check in the sampler, but different samplers would need to be checked differently. I’ll see about adding a “num_cuts” attribute.